<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профи MAX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/about/icons/logo.png</normaloff>:/about/icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="TabWidgetMDI" name="tabWidgetMDI">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Фаил</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionOpenReport"/>
    <addaction name="actionSaveReport"/>
    <addaction name="separator"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExportPDF"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="actionPref"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuService"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionOpenReport"/>
   <addaction name="actionSaveReport"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExportPDF"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSprav">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Главная</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsSprav">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="toolBoxSprav">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>О Программе</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/launch.png</normaloff>:/about/icons/launch.png</iconset>
   </property>
   <property name="text">
    <string>Соединение...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/exit.png</normaloff>:/about/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/editcut.png</normaloff>:/about/icons/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/editcopy.png</normaloff>:/about/icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/editpaste.png</normaloff>:/about/icons/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/import.png</normaloff>:/about/icons/import.png</iconset>
   </property>
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/export.png</normaloff>:/about/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Выделить все</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/editundo.png</normaloff>:/about/icons/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/editredo.png</normaloff>:/about/icons/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Вернуть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSwitchPPO">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/ppo_mini.png</normaloff>:/about/icons/ppo_mini.png</iconset>
   </property>
   <property name="text">
    <string>Выбор ППО</string>
   </property>
  </action>
  <action name="actionReportGeneral">
   <property name="text">
    <string>Общие сведения</string>
   </property>
  </action>
  <action name="actionReportForm1E">
   <property name="text">
    <string>Форма 1Э</string>
   </property>
  </action>
  <action name="actionReportForm2">
   <property name="text">
    <string>Форма 2</string>
   </property>
  </action>
  <action name="actionReportForm4">
   <property name="text">
    <string>Отчеты и выборы</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/fileprint.png</normaloff>:/about/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Печать...</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/fileprint.png</normaloff>:/about/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Просмотр печати</string>
   </property>
   <property name="iconText">
    <string>Просмотр печати...</string>
   </property>
  </action>
  <action name="actionExportPDF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/exportpdf.png</normaloff>:/about/icons/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт в PDF...</string>
   </property>
  </action>
  <action name="actionSaveReport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/filesave.png</normaloff>:/about/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить отчет</string>
   </property>
  </action>
  <action name="actionOpenReport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/fileopen.png</normaloff>:/about/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Открыть отчет</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="actionPref">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/about/icons/kservices.png</normaloff>:/about/icons/kservices.png</iconset>
   </property>
   <property name="text">
    <string>Настройка</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidgetMDI</class>
   <extends>QTabWidget</extends>
   <header>tabwidgetmdi.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
